heft-plugins/heft-webpack4-plugin/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "peerDependencies": {
     "@rushstack/heft": "^0.45.6",
+    "@types/webpack": "4.41.24",
```

Comment:
Also:
```
"peerDependenciesMeta": {
  "@types/webpack": {
    "optional": true
  }
},
```

Reply:
Also, this should be some form of range; we don't want to impose exact versions in a peer dependency

Reply:
Fixed. Using ^4 as the type range